velox/functions/sparksql/String.h

```diff
@@ -94,6 +94,59 @@ struct Sha1HexStringFunction {
   }
 };
 
+/// sha2 function
+/// sha2(varbinary, bitLength) -> string
+/// Calculate SHA-2 family of functions (SHA-224, SHA-256,
+/// SHA-384, and SHA-512) and convert the result to a hex string.
+/// Returns SHA-2 digest as hex string.
+/// If asking for an unsupported SHA function, the return value is NULL.
+template <typename T>
+struct Sha2HexStringFunction {
```

Comment:
Why not just Sha2Function

Reply:
The spark sha2 function converts the result to a base16 string, in addition to calculating the result

Reply:
Thank you for clariying.